frontend/angular.json

```diff
@@ -80,6 +80,47 @@
                 }
               ]
             },
+            "prod": {
```

Comment:
there's already a `prod` build definition above that looks the same as this, but labeled `production`. The values all look the same otherwise

Reply:
Our build CI expects environment names to be prod,qa,staging. If its not an issue, I can delete the `production` config since it wouldnt be used.